### ISSUE TITLE
Is there any way to run MediaPipe in my own web app today?

### ISSUE THREAD

**Issue by siddharthoblix** [google/mediapipe#1206]
Is there any way to run MediaPipe in my own web app today?

The [primary MediaPipe GitHub page](https://github.com/google/mediapipe) links to this page: [MediaPipe on the Web](https://developers.googleblog.com/2020/01/mediapipe-on-web.html), which, in turn, links to this page: [MediaPipe graphs](https://mediapipe.readthedocs.io/en/latest/concepts.html#graph), which is broken.

Has anyone been able to successfully integrate MediaPipe into their own web app?

**Comment by droidlabour** [comment]
@siddharthoblix Its possible. You can try out the demo at [https://viz.mediapipe.dev/demo/pose_tracking](https://viz.mediapipe.dev/demo/pose_tracking)

**Comment by mgyong** [comment]
We have not open sourced any web examples yet. You can only use the MediaPipe Viisualizer. We plan to release something shortly that helps developers to run MediaPipe in javascript

**Comment by droidlabour** [comment]
@mgyong Is it possible to shed some details about how to create the `web assembly` & the `data` file packages.
I tried to run the pose-estimation demo locally and it worked (by just copying the assets) and I believe it can be made workable if you can share details on creating those 2 files.

Thanks

**Comment by senshin998** [comment]
thanks，I’m on the edge of my seat.

**Comment by delebash** [comment]
I did the same as far as extracting the mediapipe web version but I am hoping for an official build instructions and api for the web.  Here is my repo of the extraction if interested https://github.com/delebash/mediapipe_hands_web

**Comment by mgyong** [comment]
Are there startups that are interested in using Full body pose on the web for use cases like online fitness? We are releasing MediaPipe JS solutions API shortly and like to see folks interested in this. email mgyong@google.com

**Comment by siddharthoblix** [comment]
@mgyong We ([Reactiv](https://www.reactivrehab.com)) are a start-up using hand tracking to gamify physical therapy, but our solution is built in Unity. We did some research earlier on building desktop web apps using MediaPipe (that's when I created this issue). For now, our focus is Unity, but we are likely to explore web-based solutions again in the future. In the meantime, we would be keen to hear any updates regarding a Unity integration of MediaPipe.
